packages/botonic-react/src/components/multichannel/multichannel-utils.js

```diff
@@ -78,3 +78,35 @@ export const isFacebook = context =>
   context.session &&
   context.session.user &&
   context.session.user.provider == Providers.Messaging.FACEBOOK
+
+export function whatsappMarkdown(text) {
+  const markdownRegex = /(\*\*|__)(.*?)\1|(\*|_)(.*?)\3/g
+
+  const matches = text.match(markdownRegex)
+  if (matches) {
+    const matchesResult = matches.map(match => {
+      if (match.startsWith('**')) {
```

Comment:
i would simplify it in one liner, but as you prefer. We should define a prettier/linter configuration that everyone agree